ci/ci-test.sh

```diff
@@ -107,7 +107,7 @@ runTestSuite() {
 
   echo "running ginkgo test case with coverage ${coverageFile}"
 
-  if ! ginkgo -v -coverprofile="${coverageFile}" --label-filter="${labelFilter}" -covermode=atomic; then
+  if ! ginkgo -p -v -coverprofile="${coverageFile}" --label-filter="${labelFilter}" -covermode=atomic; then
```

Comment:
Is this running it in parallel?
I'd rather run in parallel against the same cluster as can potential cause unpredictable behavior between running tests.
Maybe we can leave it entirely to another piece of work

Reply:
yes for this PR we can stick to volume and snapshot functioning

Reply:
I meant, I'd rather **not** run in parallel on the same cluster as can potential cause unpredictable behavior between running tests if they step on each others toes.

Reply:
removed `-p`